pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+## Meetings scheduling automation and related changes
+
+Because of the community's growth and the growing number of topic-specific meetings, we had to spend some time improving things, and automating as much as possible. After some initial discussions like [this one about Google-services](https://github.com/asyncapi/community/discussions/236), some initial requirements were defined, called [Meeting as a Service](https://github.com/asyncapi/community/issues/245).
+
+- A week ago, we stopped using our `asyncapi-users` Google Group. We removed it, and all the members were migrated as subscribers of the AsyncAPI Newsletter. This is the place where we will now send email notifications every Monday-morning with a list of upcoming meetings,
+- We switched to the new Google Calendar instance managed by the official AsyncAPI Google account. You should remove/unsubscribe from the old calendar and [subscribe to the new one](https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t),
+- [Our Technical Steering Committee (TSC)](https://github.com/asyncapi/community/discussions/295) approved to sponsor 8 Zoom licenses to host official AsyncAPI meetings. So far, only 3 licenses have been assigned. We have lots of room for more meetings and hosts, if there is a need.
+
+Huge thanks to [Krishna Kumar](https://github.com/Krishks369) and [Shreyansh Jain](https://github.com/sudoshreyansh) that helped in research related to the implementation :heart:
+
+If you want to learn more about becoming a host of the meeting, or other things related to meetings, get familiar with our [dedicated meetings FAQ](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).
+
+Oh, and I almost forgot, look at this :point_down: :star: 
+All [info about AsyncAPI-related meetings](https://www.asyncapi.com/community/meetings) in one beatifull place :heart:
+
+<Figure
+  src="/img/posts/2022Q1-summary/meetings.webp"
+  className="text-center"
+  caption="View of dedicated community/meetings page on the AsyncAPI website."
+/>
+
+## Upcoming 2.4 and 3.0 releases of the spec
+
+We released a new [2.4.0 version of AsyncAPI Specification](https://github.com/asyncapi/spec/releases/tag/v2.4.0). There are many good features, like the possibility to provide the `id` of the Message unique for the entire AsyncAPI file or the ability to specify security valid only for a given operation. You definitely need to read [release notes](https://www.asyncapi.com/blog/release-notes-2.4.0).
+
+Huge shout out to [Sergio Moya](https://twitter.com/smoyac), who was a release coordinator for 2.4 release and pushed all the doors to make sure the release could go out.
+
+2.4.0 was released, but this doesn't block us from working on something much bigger, 3.0.0 version of the specification :rocket:
```

Comment:
```suggestion
The release of 2.4.0 helps us continue working toward something much bigger; the 3.0.0 version of the specification! :rocket:
```

Reply:
2.4 and 3.0 are not related, so 2.4 doesn't "help" working on 3.0

I just wanted to highlight that regular 2.x releases do not mean that we do not think on more significant changes that will come in 3.0

Reply:
The reason why I use the word help is because now that you are done working on one project you have more time to do the next one ;)

I assure you it was not because one project relied on the other ✨😻🥳

Reply:
> help is because now that you are done working on one project you have more time to do the next one

but that is the thing, both are super not related, for example, Jonas works on 3.0 and has basically 0 to do with 2.4

Reply:
I hear you; you're telling me that you have a concern that the wording will sound like they are related.

Totally get it, you can disregard this one if you like!

Reply:
oh but still **add a punctuation mark at the end** tho, that was missing :)

Reply:
done, `!` added, thanks 🙏🏼